config/environments/production.rb

```diff
@@ -19,7 +19,7 @@
   config.public_file_server.enabled = ENV['RAILS_SERVE_STATIC_FILES'].present?
 
   # Compress JavaScripts and CSS.
-  config.assets.js_compressor = :uglifier
+  # config.assets.js_compressor = :uglifier DOES NOT WORK ON HEROKU
```

Comment:
I think it would work to either just disable uglifier here or just add the uglifier gem; I think `:uglifier` refers to the uglifier gem.